### DISCUSSION TITLE
Compatibility with IP150+ module

### DISCUSSION THREAD

**Discussion by eduardoboucas** [ParadoxAlarmInterface/pai#509] [Q&A]
Hi all. I have a Paradox alarm system. I'd like to integrate it with Home Assistant, so I'm thinking of buying the IP150 module specifically for this purpose.

However, from what I gather the IP150 module has been deprecated in favour of the newer IP150+, and from what I've been reading this newer version is harder to integrate, and likely involves using the cloud features that suffer from [this vulnerability](https://github.com/CriticalSecurity/paradox).

Can anyone help me understand if that's the case? If I'm looking to integrate the system with Home Assistant, what is the easiest path in terms of new hardware to buy?

Many thanks in advance!

**Comment by yozik04** [comment]
New IP150+ are bad. People report problems with them periodically. Cloud way is not that stable as it can be with local connection.
